### PR TITLE
style(switch): update fluent switch styles

### LINKED ISSUE #11130: Switch in Fluent Theme
## Is your feature request related to a problem? Please describe.
The Switch does not have defined style for hover state when the Switch is on/selected.

## Describe the solution you'd like
The on/selected Switch has a different background color when hovered.

### PR DESCRIPTION
**Pull request by didimmova**
Closes #11130 

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_switch.scss [hunk 1]
```diff
@@ -121,6 +121,10 @@ $fluent-switch: extend(
             color: ('grays', 800),
         ),
 
+        track-on-hover-color:(
+            color: ('primary', 900)
```
Comment (StefanIvanov) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_switch.scss:125]:
```suggestion
            color: ('primary', 800)
```

Comment (StefanIvanov) [reply projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_switch.scss:125]: also update the border-on-hover-color: to ('primary', 800)

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_switch.scss [hunk 2]
```diff
@@ -48,6 +48,10 @@ $dark-fluent-switch: extend($fluent-switch, $base-dark-switch, (
         color: ('grays', 200)
     ),
 
+    track-on-hover-color:(
+        color: ('primary', 100)
```
Comment (StefanIvanov) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_switch.scss:52]:
```suggestion
        color: ('primary', 200)
```

Comment (StefanIvanov) [reply projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_switch.scss:52]: also update the border-on-hover-color: to ('primary', 200)

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Comment by StefanIvanov** [comment]
@didimmova you noticed the comments I left to update the border-on-hover-color: as well, didn't you? They should use the same colors as track-on-hover-color: for the respective theme variants

**Review by simeonoff — DISMISSED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Pull request closed without merging**